Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:latest AS builder
+WORKDIR /app
+COPY . .
+RUN go mod download
+RUN go build -o bin ./cmd/server/
+CMD ["./bin"]
```

Comment:
that's not really a good way to make an image, you'd want to make it from scratch and use go install with tags to embed version, but check goreleaser